tests/components/vicare/test_init.py

```diff
@@ -0,0 +1,33 @@
+"""Test ViCare init."""
+
+from unittest.mock import patch
+
+from homeassistant.components.vicare.const import CONF_EXTENDED_API
+from homeassistant.core import HomeAssistant
+
+from . import MODULE
+from .conftest import Fixture, MockPyViCare, mock_config_entry
+
+
+async def test_v1_v1_1_migration(
+    hass: HomeAssistant,
+    mock_v1_config_entry: mock_config_entry,
+) -> None:
+    """Test the ViCare binary sensor."""
+    fixtures: list[Fixture] = [Fixture({"type:boiler"}, "vicare/Vitodens300W.json")]
+    with patch(
+        f"{MODULE}.vicare_login",
+        return_value=MockPyViCare(fixtures),
+    ):
+        assert mock_v1_config_entry.version == 1
+        assert mock_v1_config_entry.minor_version == 0
```

Comment:
@frenck is my assumption correct that `minor_version` is currently `0` or does it start with `1`?

Reply:
I guess it's `1`, so migration function need to respect that..
https://github.com/home-assistant/core/blob/e30f3155650159e15b743aea8ff751849236f0bd/homeassistant/config_entries.py#L1614